.github/workflows/release.yml

```diff
@@ -145,42 +145,24 @@ jobs:
     name: Build binaries
     needs: [release-please]
     if: ${{ needs.release-please.outputs.tag-name }}
-    runs-on: ubuntu-22.04
-    steps:
-      - name: Dispatch to publish workflow
-        uses: benc-uk/workflow-dispatch@v1
-        with:
-          workflow: publish-nargo.yml
-          repo: noir-lang/noir
-          ref: master
-          token: ${{ secrets.GITHUB_TOKEN }}
-          inputs: '{ "tag": "${{ needs.release-please.outputs.tag-name }}", "publish": true }'
+    uses: noir-lang/noir/.github/workflows/publish-nargo.yml@master
```

Comment:
Why is this pinned to `master`?

Reply:
I might be misunderstanding it: it looks like the `release` workflow [only runs on master](https://github.com/noir-lang/noir/blob/2711b8414d1e5a071bb01ad11a2d0b176e366176/.github/workflows/release.yml#L6), and while previously we dispatched via the API, which required to tell which branch to run the publish workflow on, now we call it inside the workflow, so it will run on the same branch as the release workflow itself. 

With `@master`, would it just pull the workflow definition from `master` and apply it to whatever branch this workflow is running on, or would `master` apply to the jobs in the `publish-nargo.yml` workflow itself?